node_cli/index.js

```diff
@@ -0,0 +1,59 @@
+const inquirer = require("inquirer");
+const chalk = require("chalk");
+const clear = require("clear");
+const figlet = require("figlet");
+const realmAuth = require("./realm/realmAuth");
+const main = require("./main");
+
+const output = (text, type) => {
+  if (type == "result") console.log(chalk.cyanBright(text));
+  if (type == "header") console.log(chalk.cyanBright.bold("\n" + text + "\n"));
+  if (type == "error") console.log(chalk.red.bold("\n ❗" + text + " ❗\n"));
+};
+
+let authedUser;
+
+clear();
+console.log(
+  chalk.green.bold(
+    figlet.textSync("realm-cli", {
+      font: "colossal",
+    })
+  )
+);
+
+function run() {
+  output("*** WELCOME ***", "header");
+  output("Please log in to your Realm account:", "header");
+  inquirer
```

Comment:
nit: maybe use async/await here?

```js
async function run() {
  output("*** WELCOME ***", "header");
  output("Please log in to your Realm account:", "header");
  const input = await inquirer.prompt(...)
  const user = await realmAuth.login(input.email, input.password);
  output("You have successfully logged in.", "result");
  if (!user.isLoggedIn) {
    output("Something went wrong logging you in.", "error");
    return;
  }
  output("You have successfully logged in.", "result");
  return main.mainMenu(user);
}
run().catch((err) => {
    output(err, "error");
});
```

Reply:
@nlarew I'm not opposed to changing it. I stuck with the pattern that Inquirer uses in their docs. Is there a compelling argument to change it? Readability?